### PR TITLE
Finished travel page

### PR DESCRIPTION
**Pull request by theswerd**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here:
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

## What does this PR accomplish?

Updates the Travel Info Page UI to match current Figma

## Did you add any dependencies?

No

## How did you test the change?

iOS Simulator

Simulator Screen Shot - iPhone 8 Plus - 2020-03-30 at 01 41 07
Simulator Screen Shot - iPhone 8 Plus - 2020-03-30 at 01 41 13
Simulator Screen Shot - iPhone 8 Plus - 2020-03-30 at 01 41 19

### CODE REVIEW

**Comment by theswerd** [comment]
Simulator Screen Shot - iPhone 8 Plus - 2020-03-30 at 01 47 25
When warning is hidden it now shows abbreviation

File: client/flutter/lib/pages/travel_advice.dart [hunk 1]
```diff
@@ -1,62 +1,116 @@
...
+  Widget build(
+      BuildContext context, double shrinkOffset, bool overlapsContent) {
+    return LayoutBuilder(builder: (layoutContext, constraints) {
+      return Container(
+        color: Color(0xffD82037),
+        height: constraints.maxHeight,
+        padding: EdgeInsets.symmetric(horizontal: 25, vertical: 10),
+        child: Center(
+            child: Text(
+          shrinkOffset<180?"WHO continues to advise against the application of travel or trade restrictions to countries experiencing COVID-19 outbreaks. It is prudent for travellers who are sick to delay or avoid travel to affected areas, in particular for elderly travellers and people with chronic diseases or underlying healh conditions. “Affected areas” are considered those countries, provinces, territories or cities experiencing ongoing transmission of COVID-19, in contract to areas reporting only imported cases.":"WHO travel warning",
```
Comment (advayDev1) [inline client/flutter/lib/pages/travel_advice.dart]:
```suggestion
          shrinkOffset<180?"WHO continues to advise against the application of travel or trade restrictions to countries experiencing COVID-19 outbreaks. It is prudent for travellers who are sick to delay or avoid travel to affected areas, in particular for elderly travellers and people with chronic diseases or underlying healh conditions. “Affected areas” are considered those countries, provinces, territories or cities experiencing ongoing transmission of COVID-19, in contrast to areas reporting only imported cases.":"WHO travel warning",
```

**Comment by theswerd** [comment]
@advayDev1 the sticking is intentional so that the travel warning reference always stays, I can. take it out if you want

**Comment by theswerd** [comment]
New page screenshots
Simulator Screen Shot - iPhone 8 Plus - 2020-03-30 at 12 07 47
Simulator Screen Shot - iPhone 8 Plus - 2020-03-30 at 12 07 49
Simulator Screen Shot - iPhone 8 Plus - 2020-03-30 at 12 07 53

File: client/flutter/ios/Podfile.lock [hunk 2]
```diff
@@ -55,4 +55,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 083258d7f5e80b42ea9bfee905fe93049bc04c64
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.8.4
```
Comment (advayDev1) [inline client/flutter/ios/Podfile.lock]:
```suggestion
COCOAPODS: 1.9.1
```

**Pull request merged**